exe/Daemon.hs

```diff
@@ -110,6 +111,12 @@ main = do
           Right (Right ()) -> do
             logError' "Server stopped (this should not happen)" []
             exitFailure
+  where
+    -- | Allow Content-Type header with values other then allowed by simpleCors.
+    corsWithContentType = Cors.cors (const $ Just policy)
+      where
+        policy = Cors.simpleCorsResourcePolicy
```

Comment:
There might be potential problems down the line here, but they're not relevant while there is no authentication. Can you open an issue for authentication on the web ui, and mention taking a deeper look at CORS?

Reply:
https://github.com/radicle-dev/rad-issues-frontend/issues/1